src/sdk/dataUtils.ts

```diff
@@ -224,4 +224,57 @@ export class DataUtils {
 
     return data;
   }
-}
\ No newline at end of file
+
+  /**
+  * gets supported tokens
+  * @param dto
+  * @return Promise<Token[]>
+  */
+  async getSupportedAssets(dto: GetSupportedAssetsDto): Promise<Token[]> {
+    const { chainId, provider } = await validateDto(dto, GetSupportedAssetsDto);
+
+    return this.dataModule.getSupportedAssets(chainId, provider);
+  }
+
+  /**
+  * gets quote transactions
+  * @param dto
+  * @return Promise<QuoteTransaction[]>
+  */
+  async getQuotes(dto: GetQuotesDto): Promise<QuoteTransaction[]> {
```

Comment:
Again `QuotesTransaction` doesn't make sense. Pls call it Offers or just Quotes.